### PR TITLE
Add Yellow Paint

### PR DESCRIPTION
**Pull request by Karthik99999**

### CODE REVIEW

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -1226,6 +1226,41 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+		secondary: null,
+		target: "normal",
+		type: "Ice",
+	},
+	iondeluge: {
+		inherit: true,
+		onHit() {
+			this.add(`c:|${getName('Yellow Paint')}|Paint it Yellow!`);
+		},
+	},
```
Comment (MathyFurret) [inline data/mods/ssb/moves.ts:1262]: This message can probably be moved to the innate.

**Review by HisuianZoroark — APPROVED** [review]
Looks fine but please don't directly modify regular moves just to add a quote, will merge after

File: data/mods/ssb/conditions.ts [hunk 2]
```diff
@@ -525,6 +525,18 @@ export const Conditions: {[k: string]: ModdedConditionData & {innateName?: strin
...
+		noCopy: true,
+		onStart() {
+			this.add(`c:|${getName('Yellow Paint')}|cheers`);
+		},
+		onSwitchOut() {
+			this.add(`c:|${getName('Yellow Paint')}|luckynbad`);
+		},
+		onFaint() {
+			this.add(`c:|${getName('Yellow Paint')}|The canvas is filled with different screams.`);
+		},
```
Comment (HisuianZoroark) [inline data/mods/ssb/conditions.ts:538]:
```suggestion
		},
		onModifyMove(move) {
			if (move.id === 'iondeluge') {
				move.onHit = function () {
					this.add(`c:|${getName('Yellow Paint')}|Paint it Yellow!`);
				};
			}
		},
```

File: data/mods/ssb/moves.ts (hunk 1 above)
Comment (HisuianZoroark) [inline data/mods/ssb/moves.ts:1262]:
```suggestion
```

**Pull request merged**